aspdotnet/requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+mock==2.0.0
+pytest
+pypiwin32
```

Comment:
Is this needed in the requirements as well or only for testing?

Reply:
pypiwin32 is required, but we install it separately as part of the build..

Reply:
ah makes sense it's in the core agent?